test/functional/test_framework/messages.py

```diff
@@ -1212,7 +1212,7 @@ def deserialize(self, f):
         self.block.deserialize(f)
 
     def serialize(self):
-        return self.block.serialize(with_witness=False)
+        return self.block.serialize(with_witness=True)
```

Comment:
I agree with Alex that this parameter seems to make no sense anymore, so we could change the `serialize` signature as well.